executor/linux/step.go

```diff
@@ -139,7 +139,12 @@ func (c *client) PlanStep(ctx context.Context, ctn *pipeline.Container) error {
 func (c *client) ExecStep(ctx context.Context, ctn *pipeline.Container) error {
 	b := c.build
 	r := c.repo
-	l := c.stepLogs[ctn.ID]
+
+	result, ok := c.stepLogs.Load(ctn.ID)
+	if !ok {
+		return fmt.Errorf("unable to get log from client")
```

Comment:
Potentially something similar to above?

i.e. `unable to get step log from client`

Reply:
Same as above. 😄